examples/01_plotting/brainpedia_surface.py

```diff
@@ -0,0 +1,17 @@
+from matplotlib import pyplot as plt
```

Comment:
You need a docstring with a title, for the gallery. Sphinx-gallery will not accept to build a gallery item from this.